places/views.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+from django.http.response import HttpResponse, JsonResponse
+from django.shortcuts     import render
+from django.views         import View
+from places.models        import Place
```

Comment:
1. 사용하지 않는 모듈은 지워주세요.
2. import 문 컨벤션 확인해서 맞춰주세요!
```suggestion
import json

from django.http.response import HttpResponse, JsonResponse
from django.views         import View

from places.models        import Place
```